seed/challenges/html5-and-css.json

```diff
@@ -3814,8 +3814,8 @@
         "Let's turn our <code>body</code> element's background-color white by changing its hex code to <code>#FFFFFF</code>"
       ],
       "tests": [
-        "assert($(\"body\").css(\"background-color\") === \"rgb(255, 255, 255)\", 'message: Your <code>body</code> element should have the <code>background-color</code> of white.');",
-        "assert(code.match(/#FFFFFF/ig) && code.match(/#FFFFFF/ig).length > 0, 'message: Use the <code>hex code</code> for the color white instead of the word <code>white</code>. For example <code>body { color&#58; #FFFFFF; }</code>');"
+        "assert($(\"body\").css(\"background-color\") === \"rgb(255, 255, 255)\" && code.match(/body\\s*{[\\s\\S]*?background.*[\\s\\S]*?}/gi), 'message: Your <code>body</code> element should have the <code>background-color</code> of white.');",
```

Comment:
I intentionally left that in so the first test wouldn't pass automaticly, even if there was no code written. Because the default background is allready white.

Shall i remove this?